### ISSUE TITLE
Fix FileNotFoundError

### ISSUE THREAD

**Issue by kkirsche** [kkirsche/ansible-generator#2]
Reported via sentry

```
FileNotFoundError: [Errno 2] No such file or directory: '/run/user/REDACTED/Ansible/dev/inventories/caca'
  File "ansible_generator/files.py", line 85, in touch
    with open(filename, "a") as f:
```

**Comment by kkirsche** [comment]
Caused when a user puts in `inventories/` or something like that as a prefix

**Comment by kkirsche** [comment]
Fixed in 2.1.4
